### ISSUE TITLE
When training on different datasets, do hyperparameters such as learning rate and batch size need to be adjusted specifically? Are there any verified recommended settings for reference?

### ISSUE THREAD

**Issue by H-Tenets** [LianpoWang/MaskDICNet#29]
When training on different datasets, do hyperparameters such as learning rate and batch size need to be adjusted specifically? Are there any verified recommended settings for reference?
